### DISCUSSION TITLE
Sonoff Zigbee stick +20 dbm firmware

### DISCUSSION THREAD

**Discussion by host-felice** [Koenkk/zigbee2mqtt#14751] [Devices]
Hello. I read a lot about it and I found out that adding transmit_power: 20 should do the job for the coordinator. What about the router firmware? I use other sonoff zigbee stick as routers.
Thanks!

**Comment by Koenkk** [comment]
I will release this soon, keep an eye on https://github.com/Koenkk/Z-Stack-firmware/issues/341

**Comment by host-felice** [comment]
oh! I didn't think to search with the chip name. Sorry! Thanks for the link and your great work!

**Comment by PH89** [comment]
Any updates here ? 😇

**Reply by host-felice** [reply]
> Any updates here ? 😇

https://github.com/Koenkk/zigbee2mqtt/issues/15912#issuecomment-1383991790
